roles/dosomething.base/handlers/main.yml

```diff
@@ -4,3 +4,7 @@
 - name: restart ntp
   sudo: yes
   service: name=ntp state=restarted
+
+- name: restart server
+  sudo: yes
+  shell: /sbin/reboot 
```

Comment:
True, but I like to put in full pathing for shell commands, especially
since Ansible use /bin/sh as the remote shell runner.

On Mon, May 11, 2015 at 1:40 PM, Sergii Tkachenko <notifications@github.com>
wrote:

> In roles/dosomething.base/handlers/main.yml
> <https://github.com/DoSomething/tower/pull/9#discussion_r30062475>:
>
> > @@ -4,3 +4,7 @@
> >  - name: restart ntp
> >    sudo: yes
> >    service: name=ntp state=restarted
> > +
> > +- name: restart server
> > +  sudo: yes
> > +  shell: /sbin/reboot
>
> Should work without /sbin/ prefix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DoSomething/tower/pull/9/files#r30062475>.
>